### PR TITLE
travis use tox

### PR DESCRIPTION
**Pull request by gawel**
see #43

### CODE REVIEW

**Comment by andriisoldatenko** [comment]
@gawel LGTM.

**Comment by gawel** [comment]
Not really... Las build on travis is 8 monthes old. Currently trying to swith to the organisation account and make it test PRs...

**Comment by gawel** [comment]
The organisation do not show in my travis account. The "sync" button do not solve the problem. So this will wait until someone fix it. Maybe it will appears on my account in a few days...

**Comment by gawel** [comment]
Deleted / created the service on github and... it's working.

File: .travis.yml [hunk 1]
```diff
@@ -1,57 +1,21 @@
...
+  - TOXENV=dj12
+  - TOXENV=dj13
+  - TOXENV=dj14
+  - TOXENV=dj15
+  - TOXENV=dj16
+  - TOXENV=dj15-py32
+  - TOXENV=dj15-py33
+  - TOXENV=dj15-py33-postgres
+  - TOXENV=py26-wtx1
+  - TOXENV=py27-clocale
```
Comment (andriisoldatenko) [inline .travis.yml]: ERROR: unknown environment 'py26-wtx1'

**Comment by andriisoldatenko** [comment]
@gawel  Do you need help with fixing fails?

**Comment by gawel** [comment]
You can have a look at the postgres problem if you want. I'm not familiar with it or maybe we can just drop it... dont know why it exists

**Comment by kmike** [comment]
@gawel if I recall it properly, PostgreSQL transaction manager is not the same as sqlite, and there were django-webtest bugs in past related to that, so I added an environment for running tests against PostgreSQL. But this requires some setup - see https://github.com/django-webtest/django-webtest/blob/788ae1756c3a5259393c740e0ca1a2b2d83006ec/tox.ini#L93, so tests were disabled on Travis. It should be possible to setup PostgreSQL on Travis (https://docs.travis-ci.com/user/using-postgresql/), it may require some work.

**Comment by gawel** [comment]
@kmike ok. tox config now use generated env. need to figure out which failures are ok (incompatible dj version vs py version)

**Comment by gawel** [comment]
Everything is green. Is this setup ok for you? I've dropped webtest 1.x and py33 tests

**Comment by kmike** [comment]
:+1:

**Comment by andriisoldatenko** [comment]
@gawel looks good

**Pull request merged**